app/views/users/two_factor_authentication_setup/index.html.erb

```diff
@@ -0,0 +1,30 @@
+<% if !@presenter.no_factors_enabled? %>
```

Comment:
Looks like some of the indentation may be funky here

Reply:
Updated with push after working with Shopify erb-lint.  Let me know what you think